CoffeeChat/Controller/MatchViewController.swift

```diff
@@ -8,63 +8,76 @@
 import GoogleSignIn
 import UIKit
 
+/// If the local stored matchID matches the current match from backend, then the user has already reached out
+private func userAlreadyReachedOut(to match: Match) -> Bool {
```

Comment:
Note: we may want this to eventually persist in the backend? Realized UserDefaults isn't best place to be storing info about the user, especially the "dynamic" info

Reply:
After you read this, can you let me know your thoughts or maybe just make an issue in GitHub if I am not mistaken?

Reply:
yea, I was talking to Gonzalo about this a couple days ago, but at the moment the matching object doesn't  store who already responded, and who still needs to respond, so I was using user defaults as kind of a hack way to track it. Moving forward though once the backend matching object changes we should def change/delete this